### PR TITLE
Issue 1478: Bug - No char limit on no other employees textarea

### LINKED ISSUE #1478: "No other employees" request page textbox does not have character limit
### Current Behavior

1. The "No other employees from your organization?" (no_other_contacts/) request page's textbox does not currently have a character limit nor a warning message for going over 1000 characters as seen in other pages. Demo:

video1981264903

2. Related, it is also missing a character counter for the textbox, as shown here: 

noOtherEmployees

### Expected Behavior

Like in other pages with text boxes ("Purpose" or "Anything else?"), if user surpasses character limit, there should be an error message (shown below) and user would not be able to progress: 
video1045926427

There should be a character limit and counter under the textbox:
purpose

### Steps to Reproduce

Visit the /no_other_contacts page of the request flow

### Environment

_No response_

### Additional Context

_No response_

### Issue Links

Can be impacted by... 
- #661 
- #1480

### PR DESCRIPTION
**Pull request by rachidatecs**
## Ticket

Resolves #1478 

## Changes

- Add char limit validator in view
- Edit markup

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [x] Checked that the design translated visually
- [x] Checked behavior
- [x] Checked different states (empty, one, some, error)
- [x] Checked for landmarks, page heading structure, and links
- [x] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [x] Checked keyboard navigability
- [x] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [x] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

File: src/registrar/templates/application_no_other_contacts.html [hunk 1]
```diff
@@ -2,5 +2,7 @@
 {% load static field_helpers %}
 
 {% block form_fields %}
-  {% input_with_errors forms.0.no_other_contacts_rationale %}
+  {% with attr_maxlength=1000 %}
+    {% input_with_errors forms.0.no_other_contacts_rationale %}
+  {% endwith %}
```
Comment (zandercymatics) [inline src/registrar/templates/application_no_other_contacts.html:7]: I can understand why but its a little weird django doesn't handle this for us

**Review by zandercymatics — APPROVED** [review]
Approving, looks good. Found two UI issues which may not end up being issues, really depends on design input here.

Found a small bug with the validation, we seem to be getting a double error message here. Depends on design input for that as I am not sure if we want this or not.
image

I also noticed a small inconsistency with the voiceover but I do not know if its an issue. For the top error message it just reads out the error, but the validation message reads out the error + adding the word "alert" at the end. 
image

**Comment by rachidatecs** [comment]
You're picking up on the duplication of error msgs at the form level + at the inline level. That's a feature throughout the app.

**Review by gabydisarli — APPROVED** [review]
LGTM!

**Pull request merged**